data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -145,6 +145,18 @@ datasets:
         field_type: bool
         include_in_tiles: false
         include_in_downloadable_files: false
+  - long_name: "Abandoned Mine Land Inventory System"
+    short_name: "eAMLIS"
+    module_name: "eamlis"
+    load_fields:
+      - short_name: "has_amlis"
```

Comment:
nit: "has_aml"

Reply:
Good call -- done in 702804e39047697fd3673a64fdd56256dfd82d58